hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java

```diff
@@ -614,8 +614,7 @@ public static String format(List<String> nodes) {
     for (String node : nodes) {
       String[] x = node.split(":");
       sb.append(String
-          .format("{ HostName : %s, Ratis Port : %s, Role : %s } ", x[0], x[1],
-              x[2]));
+          .format("{ HostName : %s, Ratis Port : %s } ", x[0], x[1]));
```

Comment:
Thanks @sadanand48 for reporting this issue. 

This string format is not only used on Web UI, but also the CLI scm roles command.  So I would suggest keep the Role info here.

Reply:
@ChenSammi Thanks for the review. The format method here is only used in StorageContainerManager#getRatisRoles which is an implementation of the SCMMXBean (Web UI)
In case of the ozone admin scm getroles command, it creates an SCMClient and calls SCMClient#getRatisRoles which creates an scm client and does an RPC which is executed only by leader.
The SCMContext of an SCM only knows whether itself is a leader or not, If its not a leader, then we don’t know which one is the leader without doing an RPC.